dataland-backend/backendOpenApi.json

```diff
@@ -2847,6 +2847,16 @@
         }
       },
       "LksgGeneral": {
+        "required": [
+          "dataDate",
+          "listOfProductionSites",
+          "lksgInScope",
+          "numberOfEmployees",
```

Comment:
According to Jira: 
* Display errors messages on the bar (discuss this with UX/UI)

This is not yet implemented

Reply:
ok